src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,103 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver driver)
+    {
+        super(driver);
+    }
+    @FindBy(css ="li a.songs")
+    WebElement allSongsBtn;
+    @FindBy(css = "[data-testid='sound-bar-play']")
+    WebElement manaBar;
+    @FindBy(css ="[data-testid='play-btn']")
+    WebElement playBtn;
+    @FindBy(css ="side.player-controls")
+    WebElement controlPanel;
+    @FindBy(css ="[title='Create a new playlist']")
+    WebElement newRitual;
+    @FindBy(css ="[data-testid='playlist-context-menu-create-simple']")
+    WebElement createScroll;
+    @FindBy(css ="[name='name']")
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```

For example

![POM_vs_Factory_1](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/37a6856c-d849-4eea-b747-6fff5290b504)

![POM_vs_Factory_2](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/8f2260a3-a001-4431-8087-86232e893398)